src/cil.ml

```diff
@@ -6746,7 +6751,7 @@ let rec xform_switch_stmt s break_dest cont_dest = begin
        * label_break: ; // break_stmt
        *
        * The default case, if present, must be used only if *all*
-       * non-default cases fail [ISO/IEC 9899:1999, �6.8.4.2, �5]. As
+       * non-default cases fail [ISO/IEC 9899:1999, �6.8.4.2, �5]. As
```

Comment:
My browser doesn't seem to have any glyphs for these characters, whereas the old ones were fine. Why the change?